server/routes/index.js

```diff
@@ -1290,4 +1292,73 @@ router.route('/charts/new')
       return res.status(500).send({ message: err.message })
     }
 })
+
+/**
+ * Study Details
+ */
+router.route('/study-details')
+  .get(ensureAuthenticated, async (req, res) => {
+    try {
+      const { display_name, role, icon } = req.session;
+      return res.status(200).send(studyDetailsPage({
```

Comment:
The `studyDetailsPage` takes a `user` attribute, would that be as simple as calling `req.user`? Not sure what is actually stored in `req.user`.

```suggestion
      return res.status(200).send(studyDetailsPage(req.user))
```

Reply:
The uid is stored in req.user.
![Uploading Screen Shot 2022-06-27 at 2.37.24 PM.png…]()

Reply:
👍 gotcha.